beebop/app.py

```diff
@@ -70,57 +62,83 @@ def check_connection(redis) -> None:
         abort(500, description="Redis not found")
 
 
-def generate_zip(fs: PoppunkFileStore,
-                 p_hash: str,
-                 type: str,
-                 cluster: str) -> BytesIO:
+def generate_zip(
+    fs: PoppunkFileStore, p_hash: str, type: str, cluster: str
+) -> BytesIO:
     """
     [This generates a .zip folder with results data.]
 
     :param fs: [PoppunkFileStore with path to folder to be zipped]
     :param p_hash: [project hash]
     :param type: [can be either 'microreact' or 'network']
-    :param cluster: [only relevant for 'network', since there are multiple
-        component files stored in the folder, but only the right one should
-        be included in the zip folder. For 'microreact' this can be None, as
-        the cluster information is already included in the path]
+    :param cluster: [cluster assigned]
     :return BytesIO: [memory file]
     """
     memory_file = BytesIO()
     cluster_num = get_cluster_num(cluster)
-    if type == 'microreact':
-        path_folder = fs.output_microreact(p_hash, cluster_num)
-        add_files(memory_file, path_folder)
-    elif type == 'network':
-        path_folder = fs.output_network(p_hash)
-        file_list = (f'network_component_{cluster_num}.graphml',
-                     'network_cytoscape.csv')
-        add_files(memory_file, path_folder, file_list)
+    visualisations_folder = fs.output_visualisations(p_hash, cluster_num)
+    network_files = get_network_files_for_zip(
+        visualisations_folder, cluster_num
+    )
+
+    if type == "microreact":
+        add_files(
+            memory_file, visualisations_folder, network_files, exclude=True
```

Comment:
I think just a brief comment here would make this clearer, something like:
```suggestion
            # microreact zip should include all files from the visualisations folder except those which are   
            # network files, hence set exclude to True
            memory_file, visualisations_folder, network_files, exclude=True
```